ainstein_radar_drivers/include/ainstein_radar_drivers/types_o79.h

```diff
@@ -10,7 +10,8 @@ typedef enum radar_message_type {
   ground_cartesian = 5,
   raw_sphere_16bit_pwr = 6,
   alarm_status = 7,
-  filtered_point_cloud = 8
+  filtered_point_cloud = 8,
+  tracked_cartesian_CAN = 9
```

Comment:
can you update this type name to match the firmware?

Reply:
Updated